.github/workflows/update.yml

```diff
@@ -23,4 +23,4 @@ jobs:
           commit-message: Update
           branch: automated-update
           branch-suffix: timestamp
-          title: Update
+          title: Update globals
```

Comment:
I saw you use [`Update globals`](https://github.com/sindresorhus/globals/releases/tag/v15.7.0) in the release note, so you may want do this change,

Reply:
BTW, do you want the automated update to run less frequently? I can change it to monthly if you agree.

Reply:
> BTW, do you want the automated update to run less frequently? I can change it to monthly if you agree.

👍